### ISSUE TITLE
Remove 'call for proposals' links on site

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/2016.Mozillafestival.org#76]
Seek and destroy all CFP links 💣 

1) Remove 'Call for Proposals' link in header nav bar

2) Remove the line "Have a great session idea? Head to the MozFest Call for Proposals page." for all entries on the [Sessions and Spaces page](https://mozillafestival.org/spaces-and-sessions). The last 3 entries have the line worked into a paragraph. Please remove the entire paragraph.

3) CTA on landing page (this has its own ticket #75)

cc @Saallen @mmmavis

**Comment by Saallen** [comment]
Just closed the follow forms
- [x] Call for Arabic proposals
- [x] Call for Spanish proposals
- [x] Call for French proposals

🙇 

Just waiting on main form to change URL and some text edits on the main website 💣

**Comment by alanmoo** [comment]
There's another 50 minutes till 21:00UTC...

**Comment by Saallen** [comment]
PANTS!
Why do I always mix up UTC and CET, thinking they are the same thing....
All open again- thanks @alanmoo 👍

**Comment by alanmoo** [comment]
I've deployed these changes to production, they'll kick in as the CDN propagates.
